Dockerfile

```diff
@@ -2,8 +2,11 @@ FROM python:3
 ENV PYTHONUNBUFFERED 1
 WORKDIR /django
 COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+COPY load_db.sh ./
 COPY . .
-EXPOSE 8000
+RUN pip install --no-cache-dir -r requirements.txt 
+RUN python manage.py makemigrations 
 RUN python manage.py migrate
+RUN ./load_db.sh
```

Comment:
I think the bash script should be made executable before running it
RUN chmod +x load_db.sh ??

Reply:
There's no need to make the script file executable since it already existed inside the development file system. The **COPY** command will keep the file executable as it was.